DOC-4579 RDI: improve hardware sizing info (#904)
redis/docs: content/integrate/redis-data-integration/installation/install-vm.md

will not work on VMs where IPv6 is disabled. This problem is solved with version 1.4.0.
{{< /note >}}

## Hardware sizing

RDI is mainly CPU and network bound. 
Each of the RDI VMs should have at least:

- **CPU**: A minimum of 4 CPU cores. You should consider adding
  2-6 extra cores on top of this if your dataset is big and you want to ingest the
  baseline snapshot as fast as possible.
- **RAM**: 2GB 
- **Disk**: 25GB, which includes the OS footprint. In particular, you should

>   expect the `/var` folder to require 7GB and the `/opt` folder to require 1GB (to

Reviewer: ```suggestion
  RDI requires  7GB in  `/var` and 1GB in `/opt` folder  (to
```
